lib/rails/generators/slim/component_generator.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "rails/generators/erb/component_generator"
+
+module Slim
+  module Generators
```

Comment:
these module names don't line up with the file locations. I'm surprised Rails doesn't bark about this.

Reply:
I had the same feeling but I've simply followed the pattern as I saw it in the erb generators from [rails](https://github.com/rails/rails/blob/master/railties/lib/rails/generators/erb/scaffold/scaffold_generator.rb) and [slim-rails](https://github.com/slim-template/slim-rails/blob/master/lib/generators/slim/scaffold/scaffold_generator.rb)